examples/larger-examples/program-analysis/domains/Belnap.flix

```diff
@@ -20,140 +18,160 @@ namespace Belnap {
+        match (e1, e2) {
+            case (Top, x)       => x
+            case (x, Top)       => x
+            case (True, True)   => True
+            case (False, False) => False
+            case _              => Bot
+        }
 
     ///
     /// The lattice height function.
     ///
 //    #nonNegative
-    pub def height(e: Belnap): BigInt = match e {
-        case Top    => 0ii
-        case True   => 1ii
-        case False  => 1ii
-        case Bot    => 2ii
-    }
+    pub def height(e: Belnap): BigInt =
+        use Belnap.{Top, True, False, Bot};
+        match e {
+            case Top    => 0ii
+            case True   => 1ii
+            case False  => 1ii
+            case Bot    => 2ii
+        }
 
     ///
     /// Abstracts a concrete boolean into the Belnap domain.
     ///
-    pub def alpha(b: Bool): Belnap = if (b) True else False
+    pub def alpha(b: Bool): Belnap = if (b) Belnap.True else Belnap.False
 
     ///
     /// Over-approximates the logical `not` operator.
     ///
 //    #safe1(x -> not x)
 //    #strict1 #monotone1
-    pub def not(e: Belnap): Belnap = match e {
-        case Bot    => Bot
-        case True   => False
-        case False  => True
-        case Top    => Top
-    }
+    pub def not(e: Belnap): Belnap =
+        use Belnap.{Top, True, False, Bot};
+        match e {
+            case Bot    => Bot
+            case True   => False
+            case False  => True
+            case Top    => Top
+        }
 
     ///
     /// Over-approximates the logical `and` operator.
     ///
 //    #safe2((x, y) -> x and y)
 //    #strict2 #monotone2 #commutative #associative
-    pub def and(e1: Belnap, e2: Belnap): Belnap = match (e1, e2) {
-        case (Bot, _)       => Bot
-        case (_, Bot)       => Bot
-        case (True, True)   => True
-        case (True, False)  => False
-        case (False, True)  => False
-        case (False, False) => False
-        case _              => Top
-    }
+    pub def and(e1: Belnap, e2: Belnap): Belnap =
+        use Belnap.{Top, True, False, Bot};
+        match (e1, e2) {
+            case (Bot, _)       => Bot
+            case (_, Bot)       => Bot
+            case (True, True)   => True
+            case (True, False)  => False
+            case (False, True)  => False
+            case (False, False) => False
+            case _              => Top
+        }
 
     ///
     /// Over-approximates the logical `or` operator.
     ///
 //    #safe2((x, y) -> x or y)
 //    #strict2 #monotone2 #commutative #associative
-    pub def or(e1: Belnap, e2: Belnap): Belnap = match (e1, e2) {
-        case (Bot, _)       => Bot
-        case (_, Bot)       => Bot
-        case (True, True)   => True
-        case (True, False)  => True
-        case (False, True)  => True
-        case (False, False) => False
-        case _              => Top
-    }
+    pub def or(e1: Belnap, e2: Belnap): Belnap =
+        use Belnap.{Top, True, False, Bot};
```

Comment:
I think its overkill to import in every function. I would just spell it out: Belnap.Bot and so forth.

Reply:
Can be done with search and replace.